landing-page/content/common/releases/release-notes.md

```diff
@@ -25,14 +25,18 @@ url: releases
 The latest version of Iceberg is [{{% icebergVersion %}}](https://github.com/apache/iceberg/releases/tag/apache-iceberg-{{% icebergVersion %}}).
 
 * [{{% icebergVersion %}} source tar.gz](https://www.apache.org/dyn/closer.cgi/iceberg/apache-iceberg-{{% icebergVersion %}}/apache-iceberg-{{% icebergVersion %}}.tar.gz) -- [signature](https://downloads.apache.org/iceberg/apache-iceberg-{{% icebergVersion %}}/apache-iceberg-{{% icebergVersion %}}.tar.gz.asc) -- [sha512](https://downloads.apache.org/iceberg/apache-iceberg-{{% icebergVersion %}}/apache-iceberg-{{% icebergVersion %}}.tar.gz.sha512)
+* [{{% icebergVersion %}} Spark 3.2 runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.2_2.12/{{% icebergVersion %}}/iceberg-spark-runtime-3.2_2.12-{{% icebergVersion %}}.jar)
+* [{{% icebergVersion %}} Spark 3.1 runtime Jar](https://search.maven.org/remotecontent?filepath=org/apache/iceberg/iceberg-spark-runtime-3.1_2.12/{{% icebergVersion %}}/iceberg-spark-runtime-3.1_2.12-{{% icebergVersion %}}.jar)
```

Comment:
Is there any plan to align the iceberg-spark runtime jar among 2.4, 3.0, 3.1 and 3.2 ? I see the spark 2.4 has the name `iceberg-spark-runtime-0.13.0`, spark 3.0 has the name `iceberg-spark3-runtime-0.13.0` but spark 3.1 or spark 3.2 has the name `iceberg-spark-runtime-3.1_2.12-0.13.0`.

Why not just name all the runtime jar as format `iceberg-spark-runtime-$sparkVersion_$scalaVersion-$icebergVersion` ?

Reply:
I don't know how important it is to keep backwards compatibility for jar names, but for now it seems that we are trying to keep 3.0 and 2.4 with old names. Any thoughts? @rdblue

Reply:
I think this jar naming issue don't block this PR, but  the inconsistent name approach for different spark versions seems increase the cost to understand which is the correct groupId/artifactId to use for the downstream users.  For my personal understanding,   I will recommend to use the similar name approach for spark 2.4 & 3.0.

Reply:
I think it's too late to do this for the 0.13.0 release. We may want to consider it for 0.14.0, but we're stuck with it for now.